contentwarehouse/snippets/create_document_sample.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+# [START contentwarehouse_create_document]
+
+from google.cloud import contentwarehouse_v1 as contentwarehouse
+
+
+def sample_create_document(
+    project_number: str,
+    location: str,
+    raw_doc_path: str,
+    mime_type: str,
+    document_schema_id: str,
+    user_id: str,
+    reference_id: str = "",
+) -> contentwarehouse.types.CreateDocumentResponse:
+    """Creates a document.
+
+    Args:
+        project_number: Google Cloud project number.
+        location: Google Cloud project location.
+        raw_doc_path: Raw document file in Cloud Storage path.
+        mime_type: Document format eg. PDF/DOCX etc.
```

Comment:
This is not just the Document format, it's a media type. `PDF/DOCX` are not valid examples. I'd recommend adding a link to the docs that provides a list of all of the supported types.